### PR TITLE
Added platform-specific tests

### PR DESCRIPTION
**Pull request by barkingfoodog**
- init_command
- run_command
- install_command

### CODE REVIEW

**Comment by barkingfoodog** [comment]
@gtmanfred here are the tests from #50 updated for #74 from @lomeroe .

**Comment by gtmanfred** [comment]
Awesome, thanks!

I will try to find time to get to testing this today, but I am off all next week so it might have to wait.

Daniel

File: spec/kitchen/provisioner/salt_solo_spec.rb [hunk 1]
```diff
@@ -113,8 +113,20 @@
...
+      end
+    end
+
+    context 'when windows' do
+      let(:platform) do
+        instance_double(Kitchen::Platform, os_type: 'windows')
+      end
+   
+      it 'should use powershell' do
+        is_expected.to match(/mkdir -Path/)
```
Comment (barkingfoodog) [inline spec/kitchen/provisioner/salt_solo_spec.rb]: @lomeroe we'll need to update this if #105 gets merged to expect `mkdir -Force -Path`

Comment (gtmanfred) [reply spec/kitchen/provisioner/salt_solo_spec.rb]: I merged it, can you go ahead and update this?

Thanks,
Daniel

**Comment by barkingfoodog** [comment]
@gtmanfred updated to this will pass against the change in #105 .

**Comment by gtmanfred** [comment]
Awesome thanks.  I think I am almost done with the stuff I am working on
today, so hopefully I will get some time to just do an extra test run on
this.

Thanks!
Daniel

**Pull request merged**